tests.sh

```diff
@@ -65,7 +65,12 @@ echo -e "\n[TEST] Docker compose version"
 docker-compose --version
 docker-compose --version | grep "docker-compose version 1.17"
 
+echo -e "\n[TEST] AWS cli version"
+export PATH=~/.local/bin:$PATH
+which aws
```

Comment:
We can't do `grep "<version>"` because aws commands don't return any output. The output to the STDOUT is side effect :D

Reply:
@bmarkons Yeah, it seems that aws logs to stderr. You can redirect stderr to stdout to get the version:

```
aws --version 2>&1 | grep "1.12"
```

This passed on my computer.

Reply:
Hah :D

Reply:
If you really want to hack, you can even swap stderr with stdout:

```
command 3>&1 1>&2 2>&3
```

Reply:
lol 😆